pkg/kafka_client/client.go

```diff
@@ -12,13 +12,25 @@ import (
 const MAX_EARLIEST int64 = 100
 
 type Options struct {
-	BootstrapServers string `json:"bootstrapServers"`
+	BootstrapServers   string `json:"bootstrapServers"`
+	SecurityProtocol   string `json:"securityProtocol"`
+	SaslMechanisms     string `json:"saslMechanisms"`
+	SaslUsername       string `json:"saslUsername"`
+	SaslPassword       string `json:"saslPassword"`
+	Debug              string `json:"debug"`
+	healthcheckTimeout int32  `json:"debug"`
```

Comment:
The `healthcheckTimeout` is not used in the code. It should be used in [here](https://github.com/hoptical/grafana-kafka-datasource/blob/54b461b391ffb8895c614bebcd6c7f8e5f568470/pkg/kafka_client/client.go#L152).